### PR TITLE
Add kernel command line argument documentation

### PR DESCRIPTION
**Pull request by austinfrey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,18 @@ Arguments:
                 Must be >= 33792 kb (~33 mb)
   --label       Label of the new image, defaults to "RUNTIMEJS"
 ```
+### Environment Variables*
+There is currently no officially supported process for passing environment variables into a runtime instance. There is a work around that will allow the passing of Kernel Command Line arguments instead.
```
Comment (facekapow) [inline README.md]: Probably a good idea to add a link to the issue on the main repo (https://github.com/runtimejs/runtime/issues/134) here (e.g. "While there is a [proposal](link-here) to add this feature, there is currently no officially...").

File: README.md [hunk 2]
```diff
@@ -101,6 +101,18 @@ Arguments:
...
+```
+runtime run --append <your env args> initrd
+```
+To access this variable in your code user the following:
+```
+const wholeString = __SYSCALL.getCommandLine();
+const arrayOfArguments = wholestring.split(' ');
+arrayOfArguments[i]; // where i is the index of your argument inthe array
```
Comment (facekapow) [inline README.md]: `inthe` -> `in the`

File: README.md [hunk 3]
```diff
@@ -101,6 +101,18 @@ Arguments:
...
+```
+*This process is subject to change
```
Comment (facekapow) [inline README.md]: I don't know about this, it feels too... legal 😄

File: README.md [hunk 4]
```diff
@@ -101,6 +101,18 @@ Arguments:
                 Must be >= 33792 kb (~33 mb)
   --label       Label of the new image, defaults to "RUNTIMEJS"
 ```
+### Environment Variables*
+There is currently no officially supported process for passing environment variables into a runtime instance. There is a work around that will allow the passing of Kernel Command Line arguments instead.
+```
+runtime run --append <your env args> initrd
+```
+To access this variable in your code user the following:
```
Comment (facekapow) [inline README.md]: `user` -> `use`

**Review by facekapow — CHANGES_REQUESTED** [review]
Looks good overall, just a few minor nitpicks/issues

File: README.md (hunk 3 above)
Comment (austinfrey) [reply README.md]: lol OK, cool. I can make the edits. Thanks for the feedback

File: README.md [hunk 5]
```diff
@@ -102,17 +102,16 @@ Arguments:
   --label       Label of the new image, defaults to "RUNTIMEJS"
 ```
 ### Environment Variables*
```
Comment (facekapow) [inline README.md]: Remove `*`?

File: README.md [hunk 6]
```diff
@@ -102,17 +102,16 @@ Arguments:
...
-There is currently no officially supported process for passing environment variables into a runtime instance. There is a work around that will allow the passing of Kernel Command Line arguments instead.
+While there is a [proposal](runtimejs/runtime/issues/134) to add envirionment variables from the command line, there is currently no officially supported process for passing them into a runtime instance. However, there is a work around that will allow the passing of Kernel Command Line arguments instead.
```
Comment (facekapow) [inline README.md]: Gotta use the full URL (`https://github.com/runtimejs/runtime/issues/134`) instead; I tried following the link from your version of the readme and it gave me a 404.

**Review by facekapow — CHANGES_REQUESTED** [review]
2 little things

**Review by facekapow — APPROVED** [review]

**Comment by iefserge** [comment]
👍

**Pull request merged**